src/common/ui-components/Video/Video.module.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../../styles/partials/variables';
+
+.container {
+  position: absolute;
+  height: 110vh;
```

Comment:
110vh seems odd, is this needed? The video itself is not taking 110vh in the current environment?

Reply:
This is outdated.